src/pages/Code.js

```diff
@@ -31,6 +34,8 @@ class Code extends Component {
             const files = content.public_files;
 
             this.setState({ files: files });
+
+            this.setState({ question: content.question });
```

Comment:
You should call this only once to avoid having to re-render: `this.setState({files: files, question: content.question})`. Also if the key and the value share the same name, you can shorten it to: `this.setState({files, question: content.question})`.

Reply:
Thanks for the feedback :)